### PR TITLE
add disabledDirs config

### PR DESCRIPTION
**Pull request by aviatesk**
corresponds to https://github.com/julia-vscode/LanguageServer.jl/pull/792

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -388,6 +388,14 @@
                     ],
                     "description": "Highlight unknown symbols. The `symbols` option will not mark unknown fields."
                 },
+                "julia.lint.disabledDirs": {
+                    "type": "array",
+                    "default": [
+                        "docs",
+                        "test"
+                    ],
+                    "markdownDescription": "Specifies sub-directories in [a package-format directory](https://docs.julialang.org/en/v1/manual/code-loading/#Package-directories-1) where only basic lintings are enabled and most false positve lintings would be surpressed."
```
Comment (pfitzseb) [inline package.json:397]:
```suggestion
                    "markdownDescription": "Specifies sub-directories in [a package directory](https://docs.julialang.org/en/v1/manual/code-loading/#Package-directories-1) where only basic linting is. This drastically lowers the chance for false positives."
```

**Comment by davidanthoff** [comment]
I'm torn on this one :) Mainly because I think we'll change all of this again in our hopefully final redesign of how we handle environments. I also think we should probably at some point have a way to configure the linter in a non VS Code way so that these linting settings also apply if one were to run the linter in a stand-alone fashion (something we also don't have yet), say as part of CI. On the other hand, maybe not so bad to break this setting again in the not too distant future, and for now just support it? Not really sure, I'll vote "neutral" and let you three haggle it out ;)

**Comment by pfitzseb** [comment]
I think I'd like to have this in for now.

**Comment by ZacLN** [comment]
I don't think this is the right approach moving forward but don't see any reason not to include it now

**Pull request merged**